.editorconfig

```diff
@@ -0,0 +1,13 @@
+# editorconfig.org
+
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+charset = utf-8
+
+[*.{js,ts,json}]
+indent_style = space
+indent_size = 2
```

Comment:
I prefer 4 spaces, but set it to 2 to match the current style.

Reply:
Since it is almost a complete rewrite anyway I wouldn't mind if you'd change it to 4.